src/hooks/useSnackMessage.ts

```diff
@@ -143,9 +143,9 @@ function displayMessageWithSnackbar(
         }
         fullMessage += message;
     }
-    enqueueSnackbar(fullMessage, {
+    return enqueueSnackbar(fullMessage, {
         variant: level,
-        persist: persistent,
+        ...enqueOption,
```

Comment:
```suggestion
        ...enqueOption,
        variant,
        style: { whiteSpace: 'pre-line' },
```
To make sure you override variant

Reply:
done